CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## Version 1.17.0
+* Release date: January 25, 2023
+* Release status: GA
+
+### What's new in 1.17.0
+* BREAKING CHANGE - Encryption is now Enabled by Default - https://github.com/microsoft/vscode-mssql/pull/17484
```

Comment:
it may be good to clarify this referring to "Connection encryption" vs. some other encryption, though perhaps that's obvious.